run.py

```diff
@@ -188,6 +195,12 @@ def main():
             else:
                 logger.error("Both parameters are need to add matched bibcodes to oracle: file path (-mf) and the source to apply (-as).")
 
+        elif args.cleanup_oracle:
+            try:
+                status = OracleUtil().cleanup_db()
```

Comment:
I would get the message returned from oracle and display it. There are three types of cleanup details are here https://github.com/adsabs/oracle_service/blob/master/oraclesrv/views.py#L382

Reply:
I refactored this to return `response.text` and modified the tests accordingly

Reply:
Matt, did not like the field `message`, just renamed it to `details` in oracle, I think makes more sense. If you would please make the change here. Also for testing, here is the unittest in oracle if you want to see what messages are brought back from cleanup https://github.com/adsabs/oracle_service/blob/master/oraclesrv/tests/unittests/test_oracle_service.py#L762.

Reply:
Great job. Thank you.